docs/Chipyard-Basics/Debugging-RTL.rst

```diff
@@ -0,0 +1,91 @@
+Debugging RTL
+======================
+
+While the packaged Chipyard configs and RTL have been tested to work,
+users will typically want to build custom chips by adding their own
+IP, or by modifying existing Chisel generators. Such changes might introduce
+bugs. This section aims to run through a typical debugging flow
+using Chipyard. We assume the user has a custom SoC configuration,
+and is trying to verify functionality by running some software test.
+
+Bringup Method
```

Comment:
This should be combined or at least reference the section on communicating with the chip.

Reply:
I think you should just remove this. I don't consider this "Basic" info.